snippets/backend/article/controller/main.ini

```diff
@@ -0,0 +1,5 @@
+[en_GB]
+article_has_been_changed = "The article has been changed in the meantime. To prevent overwriting these changes, saving the article was aborted. Please close the article and re-open it."
```

Comment:
The actual text should use "item" instead of "article", since article is Ginglish for item (or product). Keep the snippet name though.